cr.sh: line 109: latest_tag: unbound variable

cr.sh will fail if there are no tags in the repository.

Steps to recreate:

* Create an empty branch (`git checkout --orphan gh-pages`) and push to repo

```
Pushing to branch "gh-pages"
warning: not sending a push certificate since the receiving end does not support --signed push
To https://github.com/<org>/<repo>
   298a8e3..218a1fc  HEAD -> gh-pages
/__w/_actions/helm/chart-releaser-action/main/cr.sh: line 109: latest_tag: unbound variable
Error: Process completed with exit code 1.
```

I am using the latest version of the action (main branch).

--- a/cr.sh
+++ b/cr.sh
@@ -96,9 +96,12 @@ main() {
         IFS=,
         echo "${changed_charts[*]}"
       )" >changed_charts.txt
+
+      echo "chart_version=${latest_tag}" >chart_version.txt
     else
       echo "Nothing to do. No chart changes detected."
       echo "changed_charts=" >changed_charts.txt
+      echo "chart_version=" >chart_version.txt
     fi
   else
     install_chart_releaser
@@ -107,8 +110,6 @@ main() {
     release_charts
     update_index
   fi
-
-  echo "chart_version=${latest_tag}" >chart_version.txt
 
   popd >/dev/null
 }

